script.js

```diff
@@ -8,3 +8,289 @@ btnMenu.addEventListener('click', displayMenu);
+        0: 'html',
+        1: 'Bootstrap',
+        2: 'Ruby',
+      },
+      image: './Images/project3.png',
+      description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      source: '#',
+      live: '#',
+    },
+  },
+  {
+    data5: {
+      title: 'Profesional Art<br>Printing Data More',
+      mobileTitle: 'Profesional Art Printing Data',
+      tools: {
+        0: 'html',
+        1: 'Bootstrap',
+        2: 'Ruby',
+      },
+      image: './Images/project4.jpg',
+      description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      source: '#',
+      live: '#',
+    },
+  },
+  {
+    data6: {
+      title: 'Data Dashboard<br>Healthcare',
+      mobileTitle: 'Profesional Art Printing Data',
+      tools: {
+        0: 'html',
+        1: 'Bootstrap',
+        2: 'Ruby',
+      },
+      image: './Images/project5.png',
+      description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      source: '#',
+      live: '#',
+    },
+  },
+  {
+    data7: {
+      title: 'Website Protfolio',
+      mobileTitle: 'Profesional Art Printing Data',
+      tools: {
+        0: 'html',
+        1: 'Bootstrap',
+        2: 'Ruby',
+      },
+      image: './Images/project6.jpeg',
+      description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+      source: '#',
+      live: '#',
+    },
+  },
+];
+
+let restProjects = '';
+let cardNumber = 0;
+let number = 0;
+for (let i = 2; i <= 6; i += 1) {
+  number = i + 1;
+  cardNumber = `data${number}`;
+  const titleCards = allData[i][cardNumber].title;
+  const mobileTitl = allData[i][cardNumber].mobileTitle;
+  const descriptionCards = allData[i][cardNumber].description;
+  const firstTool = allData[i][cardNumber].tools[0];
+  const secondTool = allData[i][cardNumber].tools[1];
+  const thirdTool = allData[i][cardNumber].tools[2];
+
+  restProjects += `<div class="pj11" id="n${i}">
+  <div class="cont_rest">
+    <h3 class="tproj1 mobile">${mobileTitl}</h3>
+    <button class="projects desktitbut" type="button" onclick="project${number}()">
+    <h3 class="tproj1 desktop" id="h3_2">${titleCards}</h3>
+  </button>
+    <p class="desproj1">${descriptionCards}</p>
+    <ul class="media2">
+      <li>
+        <div class="cont1">${firstTool}</div>
+      </li>
+      <li>
+        <div class="cont1">${secondTool}</div>
+      </li>
+      <li>
+        <div class="cont1">${thirdTool}</div>
+      </li>
+    </ul>
+    <div class="no_button">
+      <button class="bigbutton1 button21 projects" onclick='project${number}()' type="button">See Project</button>
+    </div>
+  </div>
+</div>`;
+}
+
+const container = document.getElementById('works');
+container.innerHTML = `<div class="resent_project">
```

Comment:
- [ ]  Hi @sambeck87  please make sure the pop-up window is actually displaying as you can see I'm unable to display the pop-up when clicking over the cards.

https://user-images.githubusercontent.com/95243996/202261976-baab47c0-999c-4eff-8b5c-dd63b154ff50.mp4